content/en/docs/setup/windows/windows-networking.md

```diff
@@ -0,0 +1,305 @@
+| Preserve-Destination | Skips DNAT of service traffic, thereby preserving the virtual IP of the target service in packets reaching the backend Pod. Also disables node-node forwarding. | v1.20+ | Windows Server, version 1903 (or higher) | Set `"preserve-destination": "true"` in service annotations and enable DSR in kube-proxy. |
+| IPv4/IPv6 dual-stack networking | Native IPv4-to-IPv4 in parallel with IPv6-to-IPv6 communications to, from, and within a cluster | v1.19+ | Windows Server, version 2019 | See [IPv4/IPv6 dual-stack](#ipv4ipv6-dual-stack) |
+| Client IP preservation | Ensures that source IP of incoming ingress traffic gets preserved. Also disables node-node forwarding. | v1.20+ | Windows Server, version 2019  | Set `service.spec.externalTrafficPolicy` to "Local" and enable DSR in kube-proxy |
+{{< /table >}}
+
+### Session affinity
+
+Setting the maximum session sticky time for Windows services using
+`service.spec.sessionAffinityConfig.clientIP.timeoutSeconds` is not supported.
+
+## DNS {#dns-limitations}
+
+* ClusterFirstWithHostNet is not supported for DNS. Windows treats all names with a
+  `.` as a FQDN and skips FQDN resolution
+* On Linux, you have a DNS suffix list, which is used when trying to resolve PQDNs. On
+  Windows, you can only have 1 DNS suffix, which is the DNS suffix associated with that
+  pod's namespace (mydns.svc.cluster.local for example). Windows can resolve FQDNs
+  and services or names resolvable with just that suffix. For example, a pod spawned
+  in the default namespace, will have the DNS suffix **default.svc.cluster.local**.
+  Inside a Windows pod, you can resolve both **kubernetes.default.svc.cluster.local**
+  and **kubernetes**, but not the in-betweens, like **kubernetes.default** or
+  **kubernetes.default.svc**.
+* On Windows, there are multiple DNS resolvers that can be used. As these come with
+  slightly different behaviors, using the `Resolve-DNSName` utility for name query
+  resolutions is recommended.
+
+## IPv6 networking
+
+Kubernetes on Windows does not support single-stack "IPv6-only" networking. However,
+dual-stack IPv4/IPv6 networking for pods and nodes with single-family services
+is supported.
+
+You can use IPv4/IPv6 dual-stack networking with `l2bridge` networks. See [configure IPv4/IPv6 dual stack](/docs/concepts/services-networking/dual-stack#configure-ipv4-ipv6-dual-stack) for more details.
+
+{{< note >}}
+Overlay (VXLAN) networks on Windows do not support dual-stack networking.
+{{< /note >}}
+
+# Network troubleshooting {#troubleshooting-network}
+
+1. My Windows Pods do not have network connectivity
+
+   If you are using virtual machines, ensure that MAC spoofing is **enabled** on all
+   the VM network adapter(s).

```

Comment:
We should also make this clear somewhere in the setup section (maybe we already do?)